### PR TITLE
[trivial] add error message description

### PR DESCRIPTION
**Pull request by georgemitenkov**
## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update
- [ ] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Other (specify)

## Checklist
- [ ] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by runtian-zhou — APPROVED** [review]

**Review by vgao1996 — COMMENTED** [review]
Is this error triggerable? If not, we can land it without feature gating

**Comment by runtian-zhou** [comment]
> Is this error triggerable?

This error shouldn't be triggered as long as the VM remains single threaded.

File: third_party/move/move-vm/runtime/src/loader/mod.rs [hunk 1]
```diff
@@ -2234,10 +2234,19 @@ impl Loader {
             .write()
             .depth_formula
             .insert(name.clone(), formula.clone());
-        if prev.is_some() {
+        // TODO: remove this error or ensure we lock correctly.
```
Comment (ziaptos) [inline third_party/move/move-vm/runtime/src/loader/mod.rs:2237]: nit: Maybe a bit more description (what to lock, where to move?)

**Review by ziaptos — APPROVED** [review]

**Review by gelash — APPROVED** [review]

**Comment by georgemitenkov** [comment]
@runtian-zhou yes, it should not be, but if we have an error, the error message should give at least more info (otherwise, why have the message at all?)

**Review by ziaptos — APPROVED** [review]

**Pull request merged**